go.mod

```diff
@@ -3,13 +3,13 @@ module github.com/ipfs/go-filestore
 go 1.12
 
 require (
-	github.com/gogo/protobuf v1.3.1
```

Comment:
just FYI these are upgraded by `go mod tidy` (it upgrades protobuf, go-cid, go-blockformat)